sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ReviewSqlQueriesForSecurityVulnerabilities.cs

```diff
@@ -151,16 +151,20 @@ private void WriteUCFG<TDeclarationSyntax>(SyntaxNodeAnalysisContext context, Fu
 
             var symbol = context.SemanticModel.GetDeclaredSymbol(declaration);
 
-            if (symbol == null ||
-                symbol.IsAbstract ||
-                symbol.IsExtern ||
+            var methodSymbol = (symbol is IPropertySymbol propertySymbol)
+                ? propertySymbol.GetMethod // We are in PropertyDeclarationSyntax
+                : symbol as IMethodSymbol; // all other are methods
```

Comment:
The `GetMethod` of the `propertySymbol` was retrieved in the `MethodIdProvider` class.